frontend/simple/views/CreateGroup.vue

```diff
@@ -1,32 +1,139 @@
+                <p class="title">{{memberRemovalPercentage}}%</p>
+                <input type="range" min="0" max="100" data-vv-as="Member Removal Percentage" v-validate data-vv-rules="between:1,100" name="memberRemovalPercentage" v-model="memberRemovalPercentage">
+                <span v-show="errors.has('memberRemovalPercentage')" class="help is-danger">The Member Removal Percentage field must be between 1% and 100%.</span>
+                <!-- this is used for nightmare and only rendered in 'development' mode. Without this nightmare has trouble triggering the events for vue that updates the model when values are set -->
+                <input v-if="dev" type="text" v-model="memberRemovalPercentage" name="proxyMemberRemovalPercentage" style="opacity:0; position: absolute;">
+              </div>
+            </div>
+            <div class="tile is-4 is-vertical question-group">
+              <div class="tile notification">
+                <p class="title"><i18n>How much income will your group seek to provide</i18n></p>
+
+                <i class="fa fa-usd symbol" aria-hidden="true" ></i><input type="text" data-vv-as="Income Provided" v-validate data-vv-rules="decimal:2" name="incomeProvided" v-model="incomeProvided" class="dotted">
+                <span v-show="errors.has('incomeProvided')" class="help is-danger">The Income Provided field must be a numeric currency amount and may contain 2 decimal points.</span>
+
+              </div>
+              <div class="tile notification is-danger">
+                <p class="title"><i18n>How transparent should your group be about who contributes?</i18n></p>
+                <p class="select">
+                  <select v-validate data-vv-rules="required" data-vv-as="Conrtibution Privacy" name="contributionPrivacy" v-model="contributionPrivacy">
+                    <option value="">Select an option</option>
+                    <option value="Very Private">Very Private</option>
+                  </select>
+                </p>
+                <span v-show="errors.has('contributionPrivacy')" class="help is-danger">{{ errors.first('contributionPrivacy') }}</span>
+              </div>
+              <div class="tile">
+                <div class="center">
+                  <div id="successMsg" v-if="created" class="created">Success</div>
+                <button class="button is-success center" type="submit" :disabled="errors.any() || !fields.passed()">Create Group</button>
+                </div>
+              </div>
+            </div>
           </div>
-        </div>
+        </form>
       </div>
+      <div class="column is-1"></div>
     </div>
   </section>
 </template>
 
 <script>
+import backend from '../js/backend'
+import {Events} from '../../../shared/events'
+
 export default {
-  name: 'CreateGroupView'
+  name: 'CreateGroupView',
+  methods: {
+    submit: function () {
+      this.$validator.validateAll()
+        .then(async function () {
```

Comment:
If `.then`'s can be avoided, they should. So here plz make the `submit` function `async` and pull out this stuff up a level. i.e. call:

```js
await this.$validator.validateAll()
// stuff that was in `.then`
```